src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -148,4 +155,18 @@ private Response createUserWithSecurityGroups() {
 
         return response;
     }
+
+    @Test
+    void shouldGetSecurityGroups() throws JsonProcessingException {
+        Response response = buildRequestWithExternalGlobalAccessAuth()
+            .baseUri(getUri("/admin/security-groups"))
+            .contentType(ContentType.JSON)
+            .get()
+            .thenReturn();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<SecurityGroupWithIdAndRole> securityGroupWithIdAndRoles = objectMapper.readValue(response.asString(),
+                                                                                              new TypeReference<List<SecurityGroupWithIdAndRole>>(){});
+        assertFalse(securityGroupWithIdAndRoles.isEmpty());
```

Comment:
An assertion against the full JSON response body would add a lot of additonal value here.

Also assert that status code is 200.